session/bootstrap.go

```diff
@@ -1193,9 +1204,13 @@ func syncUpgradeState(s Session) {
 }
 
 func syncNormalRunning(s Session) {
-	_, err := ddl.ResumeAllJobsBySystem(s)
+	jobErrs, err := ddl.ResumeAllJobsBySystem(s)
 	if err != nil {
-		logutil.BgLogger().Fatal("upgrade pause all jobs failed", zap.Error(err))
+		logutil.BgLogger().Warn("[upgrading] unexpected error to resume all paused jobs: ", zap.Error(err))
+	}
+
+	for _, e := range jobErrs {
+		logutil.BgLogger().Warn("[upgrading] unable to resume the job, error: ", zap.Error(e))
```

Comment:
What would happen if some of the jobs cannot be resumed?

Reply:
Only 'paused' job can be resumed. If it can't be resumed, than its state could be pausing or others. If 'pausing', it will become 'paused' later on; otherwise, it will go as it is.